packages/cryptoplease/lib/core/balances/bl/balances_bloc.dart

```diff
@@ -21,7 +21,7 @@ part 'balances_state.dart';
 
 final _logger = Logger('BalancesBloc');
 
-@injectable
+@lazySingleton
```

Comment:
BLoCs should never be singletons. In general, relying on service locator to ensure the scope of the entity should be avoided. We use SL only as a factory, scoping should be done by providers.